hazelcast-sql/src/test/java/com/hazelcast/sql/SqlErrorTest.java

```diff
@@ -28,82 +26,13 @@
 import org.junit.runner.RunWith;
 
 import static junit.framework.TestCase.assertEquals;
-import static junit.framework.TestCase.assertTrue;
 
 /**
  * Test for different error conditions.
  */
 @RunWith(HazelcastSerialClassRunner.class)
 @Category({QuickTest.class, ParallelJVMTest.class})
 public class SqlErrorTest extends SqlErrorAbstractTest {
-    @Test
-    public void testTimeout() {
-        checkTimeout(false);
-    }
-
-    @Test
-    public void testExecutionError_fromFirstMember() {
-        checkExecutionError(false, true);
-    }
-
-    @Test
-    public void testExecutionError_fromSecondMember() {
-        checkExecutionError(false, false);
-    }
-
-    @Test
-    public void testMapMigration() {
-        checkMapMigration(false);
-    }
-
-    @Test
-    public void testMapDestroy_firstMember() {
```

Comment:
What happens now when map is destroyed during a query? Don't we want to test it?